composer.json

```diff
@@ -49,6 +49,7 @@
         "drupal/select2": "^1.13",
         "drupal/string_field_formatter": "^2.0",
         "drupal/taxonomy_machine_name": "1.x-dev",
+        "drupal/term_reference_tree": "^1.1",
```

Comment:
This is new the module that provides the taxonomy tree functionality (i.e. select all child terms automatically when a parent term is selected).